### DISCUSSION TITLE
Can you add Auto clean?

### DISCUSSION THREAD

**Discussion by MFI96** [victornpb/undiscord#344] [3. Ideas]
**Is your feature request related to a problem? Please describe.**

Working perfectly but If you need to delete more than 250 messages (Example Im deleting now 5127 message.) it starts to slow down. 

**Describe the solution you'd like**

The speed is the same when it comes to clear log. you can add automatic cleaning. May be in every 100 messages or 50 message.

**Describe alternatives you've considered**

percent view already exists. The number of messages can be displayed next to it. so no log is needed. you can offer it with a tick sign.

**Additional context**

Sincerely,

**Comment by MFI96** [comment]
and you are best ;) :)

**Comment by victornpb** [comment]
This is a great idea, It should reduce the memory usage when there's just too much stuff in the log screen. And you're right adding the count next to the percent should be fairly easy.

**Reply by 7MXek** [reply]
Would you consider a bounty on this feature? It'd be so useful to me that I'd be more than willing to donate.

**Comment by shybreakdown** [comment]
lna kürt dc ver

**Comment by 7MXek** [comment]
Is there any progress on this? I'd also love to have auto-delete functionality in this extension. Would save me from manually running a clean since I delete all of my messages after a certain period of time.

**Reply by sinnx3** [reply]
> Is there any progress on this? I'd also love to have auto-delete functionality in this extension. Would save me from manually running a clean since I delete all of my messages after a certain period of time.

You know, I was about to suggest exactly this. Personally I'd like to delete my messages everywhere (servers and DM's) that are older than 2 weeks. If we could have something that would schedule a task and compile everything necessary to do it with I'd love to have this feature. It'd bring user privacy to a whole new level on discord.
